examples/openshift/README.md

```diff
@@ -0,0 +1,45 @@
+# Install guide for OpenShift / OKD
+
+## Configuration
+Adjust the .env file in the root dir to match your needs.
+
+Some configurations are required:
+```bash
+# The base for all domains used in the template
+DOMAIN=example.com
+
+# The REST API has to be activated because it's used by the liveness and readiness probes
+JVB_ENABLE_APIS=rest
+
+# The JVB ports have to be 30000 or higher because NodePort is used for them
+JVB_PORT=30000
+JVB_TCP_PORT=30000
+
+# The secrets for all services
+# They can also be generated using the gen-passwords.sh script
+JICOFO_COMPONENT_SECRET=
+JICOFO_AUTH_PASSWORD=
+JVB_AUTH_PASSWORD=
+JIGASI_XMPP_PASSWORD=
+JIBRI_RECORDER_PASSWORD=
+JIBRI_XMPP_PASSWORD=
+```
+
+## Build
+To build the template you have to use [spiff++](https://github.com/mandelsoft/spiff). It combines all the manifests into one OpenShift Template.
+
+```console
+make build
```

Comment:
Maybe it's apparent to everyone else but I believe this needs clarification that you need to run make build from the examples/openshift sub-directory as I was first doing from the root directory :)

Reply:
I applied your suggested change. :)